diff -up k9copy-1.2.2/k9author/k9avidecode.cpp.ffmpeg k9copy-1.2.2/k9author/k9avidecode.cpp
--- k9copy-1.2.2/k9author/k9avidecode.cpp.ffmpeg	2008-10-13 07:37:35.000000000 -0500
+++ k9copy-1.2.2/k9author/k9avidecode.cpp	2008-10-13 07:37:44.000000000 -0500
@@ -10,7 +10,7 @@
 //
 //
 #include "k9avidecode.h"
-#include <ffmpeg/avcodec.h>
+#include <libavcodec/avcodec.h>
 #include <qimage.h>
 #include <dlfcn.h>
 #include <klocale.h>
diff -up k9copy-1.2.2/k9author/k9avidecode.h.ffmpeg k9copy-1.2.2/k9author/k9avidecode.h
--- k9copy-1.2.2/k9author/k9avidecode.h.ffmpeg	2007-09-29 14:12:56.000000000 -0500
+++ k9copy-1.2.2/k9author/k9avidecode.h	2008-10-13 07:35:52.000000000 -0500
@@ -14,7 +14,7 @@
 
 #include "k9common.h"
 #include <qobject.h>
-#include <ffmpeg/avformat.h>
+#include <libavformat/avformat.h>
 #include <qimage.h>
 
 /**
