diff -up k9copy-2.3.5-Source/CMakeLists.txt.dso k9copy-2.3.5-Source/CMakeLists.txt
--- k9copy-2.3.5-Source/CMakeLists.txt.dso	2010-02-07 05:30:38.000000000 -0600
+++ k9copy-2.3.5-Source/CMakeLists.txt	2010-06-13 15:33:05.618165751 -0500
@@ -325,9 +325,10 @@ kde4_add_executable(k9copy ${k9copy_SRCS
 kde4_add_executable(k9play ${k9play_SRCS})
 kde4_add_executable(k9xineplayer ${k9xineplayer_SRCS})
 
+target_link_libraries(k9copy ${KDE4_SOLID_LIBS} -ldl )
 target_link_libraries(k9copy k9copylib ${KDE4_KDEUI_LIBS} ${KDE4_KDE3SUPPORT_LIBS}  -lphonon )
-target_link_libraries(k9play k9copylib ${KDE4_KDEUI_LIBS} ${KDE4_KDE3SUPPORT_LIBS}  )
-target_link_libraries(k9xineplayer ${KDE4_KDEUI_LIBS} ${KDE4_KDE3SUPPORT_LIBS} -lxine )
+target_link_libraries(k9play k9copylib ${KDE4_KDEUI_LIBS} ${KDE4_KDE3SUPPORT_LIBS} -ldl )
+target_link_libraries(k9xineplayer ${KDE4_KDEUI_LIBS} ${KDE4_KDE3SUPPORT_LIBS} ${X11_X11_LIB} -lxine )
 
 
 install(TARGETS k9copy DESTINATION ${BIN_INSTALL_DIR} )
