diff -up k9copy-2.1.0-Source/k9copy_assistant.desktop.mimetype k9copy-2.1.0-Source/k9copy_assistant.desktop
--- k9copy-2.1.0-Source/k9copy_assistant.desktop.mimetype	2008-10-27 12:33:05.000000000 -0500
+++ k9copy-2.1.0-Source/k9copy_assistant.desktop	2008-10-30 11:35:14.000000000 -0500
@@ -9,9 +9,9 @@ Comment[tr]=DVD9 DVD5 yedekleme aracı
 GenericName=Video DVD Backup
 GenericName[fr]=Backup de DVD Video
 GenericName[tr]=DVD Vidyo Yedekleme Aracı
-MimeTypes=
+MimeType=
 Terminal=false
 Icon=k9copy
 X-DocPath=k9copy/index.html
 Categories=Qt;KDE;AudioVideo;DiscBurning;
-X-KDE-StartupNotify=true
\ No newline at end of file
+X-KDE-StartupNotify=true
diff -up k9copy-2.1.0-Source/k9copy.desktop.mimetype k9copy-2.1.0-Source/k9copy.desktop
--- k9copy-2.1.0-Source/k9copy.desktop.mimetype	2008-10-27 12:33:05.000000000 -0500
+++ k9copy-2.1.0-Source/k9copy.desktop	2008-10-30 11:34:43.000000000 -0500
@@ -9,9 +9,9 @@ Comment[tr]=DVD9 DVD5 yedekleme aracı
 GenericName=Video DVD Backup
 GenericName[fr]=Backup de DVD Video
 GenericName[tr]=DVD Vidyo Yedekleme Aracı
-MimeTypes=
+MimeType=
 Terminal=false
 Icon=k9copy
 X-DocPath=k9copy/index.html
 Categories=Qt;KDE;AudioVideo;DiscBurning;
-X-KDE-StartupNotify=true
\ No newline at end of file
+X-KDE-StartupNotify=true
